src/Libraries/RevitNodes/Elements/ReferencePlane.cs

```diff
@@ -73,7 +73,7 @@ private ReferencePlane(XYZ bubbleEnd, XYZ freeEnd, XYZ normal, View view )
         /// <param name="referencePlane"></param>
         private void InitReferencePlane(Autodesk.Revit.DB.ReferencePlane referencePlane)
         {
-            InternalReferencePlane = referencePlane;
+            InternalSetReferencePlane(referencePlane);
```

Comment:
Just curious whether this is changed because of a defect? I am asking because I remember once it is like what you have changed, but then someone changed it to be like what is before your change. But I have just looked into the code and it should be ok anyway.